src/Mesh/babylon.vertexBuffer.ts

```diff
@@ -1,14 +1,28 @@
 ﻿module BABYLON {
-    export class VertexBuffer {
-        private _mesh: Mesh;
+    export interface IVertexBuffer extends IDisposable {
+        getKind(): string;
+        getStrideSize(): number;
+        getData(): number[] | Float32Array;
+        getBuffer(): WebGLBuffer;
+        getOffset(): number;
+        getSize(): number;
+        getIsInstanced(): boolean;
+        isUpdatable(): boolean;
+        create(data?: number[] | Float32Array): void;
+        update(data: number[] | Float32Array): void;
+        updateDirectly(data: Float32Array, offset: number): void;
+    }
+
+    export class VertexBuffer implements IVertexBuffer {

```

Comment:
Not sure as `InterleavedBuffer` doesn't have a kind or implement a lot of the interface; its the `InterleavedVertexBuffer`s that map onto it that have the `kind`s and have `offset` and `size`